tcms/signals.py

```diff
@@ -209,19 +209,33 @@ def handle_comments_pre_delete(sender, **kwargs):
 def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
     """
     Send email updates to assignee after they've been
-    assigned a bug on bug creation.
+    assigned a bug on bug creation or there is a new comment to the bug.
     """
-    if not created or instance.assignee is None:
+    from tcms.core.helpers.comments import get_comments
+
+    comments = get_comments(instance)
+
+    if instance.assignee is None or comments.count() < 1:
+        return
+
+    if (not instance.status and comments.last().comment == _("*bug closed*")) or (
+        instance.status and comments.last().comment == _("*bug reopened*")
+    ):
         return
 
+    recipients = [instance.assignee.email, instance.reporter.email]
+
     from tcms.core.utils.mailto import mailto
 
     mailto(
         template_name="email/post_bug_save/email.txt",
-        recipients=[instance.assignee.email],
-        subject=_("NEW: Bug #%(pk)d - %(summary)s")
+        recipients=recipients,
+        subject=_("NOTIFICATION: Bug #%(pk)d - %(summary)s")
         % {"pk": instance.pk, "summary": instance.summary},
-        context={"bug": instance},
+        context={
+            "bug": instance,
+            "last_comment": get_comments(instance).last().comment,
```

Comment:
pass the comment as an object in case we want to use some of its other attributes in the template.